WardrobeCloset/Controllers/AccessoriesController.cs

```
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Entity;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using WardrobeCloset.Models;
+
+namespace WardrobeCloset.Controllers
+{
+    public class AccessoriesController : Controller
+    {
+        private WardrobeClosetProjectEntities db = new WardrobeClosetProjectEntities();
+
+        // GET: Accessories
+        public ActionResult Index()
+        {
+            var accessories = db.Accessories.Include(a => a.Outfit);
+            return View(accessories.ToList());
+        }
+       
+        // GET: Accessories/Details/5
+        public ActionResult Details(int? id)
+        {
+            if (id == null)
+            {
+                return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
+            }
+            Accessory accessory = db.Accessories.Find(id);
+            if (accessory == null)
```

Nice null checking! So many programmers just ignore error handling.